### DISCUSSION TITLE
Upload my own posters and fanarts

### DISCUSSION THREAD

**Discussion by Bamboozul** [trakt/showly#677] [Q&A] [answered]
I'd like to know how to upload my own, as stated by this option.

![Screenshot 2023-03-12 124831](https://user-images.githubusercontent.com/101983929/224540046-b4e43dcb-630d-46a4-9997-15ba156cb954.png)

**Comment by michaldrabik** [comment, marked as answer]
> I'd like to know how to upload my own, as stated by this option.
> 
> 
> ![Screenshot 2023-03-12 124831](https://user-images.githubusercontent.com/101983929/224540046-b4e43dcb-630d-46a4-9997-15ba156cb954.png)
> 
> 

Open show details scroll down to custom images option. Select poster or fanart and there is a cloud Icon which allows to input custom image URL.

**Reply by Bamboozul** [reply]
> > I'd like to know how to upload my own, as stated by this option.
> > ![Screenshot 2023-03-12 124831](https://user-images.githubusercontent.com/101983929/224540046-b4e43dcb-630d-46a4-9997-15ba156cb954.png)
> 
> Open show details scroll down to custom images option. Select poster or fanart and there is a cloud Icon which allows to input custom image URL.

I never thought this option was for uploading them because it says "Download Image," which apparently means save it to your app gallery.

**Reply by michaldrabik** [reply]
I guess wording could be better there, yes. It's basically getting image from external source into the app.
